x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/api/rules/coverage_overview/handle_coverage_overview_request.test.ts

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { Rule } from '@kbn/alerting-plugin/common';
+import { handleCoverageOverviewRequest } from './handle_coverage_overview_request';
+
+describe('handleCoverageOverviewRequest', () => {
+  it('processes rules in chunks', async () => {
+    const rulesClientMock = {
+      find: jest
```

Comment:
Let's reuse the rules client mock provided by the alerting framework instead of implementing a custom one. Search `rulesClientMock.create()` for examples.